src/locales/sv/install.ftl

```diff
@@ -0,0 +1,63 @@
+install-initialStep-getStarted = Get Started
+
+install-selectLanguage-stepTitleSelect = Select Language
+install-selectLanguage-selectLanguage = Select language for Coral
+install-selectLanguage-description =
+  Choose the language to be used during the installation process.
+  This will also be the default language for your Coral community.
+install-selectLanguage-language = Language
+
+install-addOrganization-stepTitle = Add Organization Details
+install-addOrganization-title = Add Organization
+install-addOrganization-description = Please tell us the name of your organization. This will appear in emails when inviting new team members.
+install-addOrganization-orgName = Organization Name
+install-addOrganization-orgNameTextField =
+   .placeholder = Organization Name
+install-addOrganization-orgEmail = Organization Contact Email
+install-addOrganization-orgEmailTextField =
+  .placeholder = Organization Contact Email
+install-addOrganization-orgURL = Organization URL
+install-addOrganization-orgURLTextField =
+  .placeholder = Organization URL
+install-addOrganization-orgURLDescription = Be sure to include <strong>http://</strong> or <strong>https://</strong> in your URL
+
+install-createYourAccount-stepTitle = Create Admin Account
+install-createYourAccount-title = Create an Administrator Account
+install-createYourAccount-email = Email
+install-createYourAccount-emailTextField =
+  .placeholder = Email
+install-createYourAccount-username = Username
+install-createYourAccount-usernameTextField =
+  .placeholder = Username
+install-createYourAccount-usernameDescription = An identifier displayed on your comments. You may use “_” and “.”
+install-createYourAccount-password = Password
+install-createYourAccount-passwordTextField =
+  .placeholder = Password
+install-createYourAccount-passwordDescription = Must be at least 8 characters
+install-createYourAccount-confirmPassword = Confirm Password
+install-createYourAccount-confirmPasswordTextField =
+  .placeholder = Confirm Password
+
+install-permittedDomains-stepTitle = Add Permitted Domains
+install-permittedDomains-title = Permitted Domains
+install-permittedDomains-description-with-scheme =
+  Enter the domains you would like to permit for { -product-name }, e.g.
+  your local, staging and production environments including the
+  scheme (ex. http://localhost:3000, https://staging.domain.com,
+  https://domain.com).
+install-permittedDomains-permittedDomains = Permitted Domains
+install-permittedDomains-permittedDomainsTextField =
+  .placeholder = Domains
+install-permittedDomains-permittedDomainsDescription = Insert domains separated by comma
+
+install-finalStep-description = Thanks for installing { -product-name }! We sent an email to verify your email address. While you finish setting up the account, you can start engaging with your readers now.
+install-finalStep-goToTheDocs = Go to the Docs
+install-finalStep-goToAdmin = Go to Admin
```

Comment:
Hey @felixdro are you able to translate this as well? If you want to do it later, just please remove this file - it always defaults to English if there are missing translations. Thank you!

Reply:
This file is now partially translated

Reply:
This file is deleted now